deployments/kubernetes/Dockerfile.istio-cni-daemonset

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:xenial
```

Comment:
use :bionic to match other istio components. Or distroless

Reply:
Using distroless.